src/main/java/uk/nhs/digital/nhsconnect/nhais/service/edifact_to_fhir/NhsNumberPatchTransactionMapper.java

```diff
@@ -0,0 +1,22 @@
+package uk.nhs.digital.nhsconnect.nhais.service.edifact_to_fhir;
+
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.Transaction;
+import uk.nhs.digital.nhsconnect.nhais.model.jsonpatch.AmendmentPatch;
+import uk.nhs.digital.nhsconnect.nhais.model.jsonpatch.JsonPatches;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class NhsNumberPatchTransactionMapper implements PatchTransactionMapper {
```

Comment:
Could you rename this to `AmendedNhsNumberPatchTransactionMapper` to avoid any confusion about present/amended?